src/components/SubmitAnswerForm.js

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from "react";
+import { Form } from "react-bootstrap";
+import styled from "styled-components";
+import { graphql } from 'babel-plugin-relay/macro'
+import { useHistory } from 'react-router-dom'
+
+import mutate from '../utils/mutate'
+
+const mutation = graphql`
+    mutation SubmitAnswerFormMutation($input: AnswerCreateInput!){
+        answerCreate(input: $input){
+            answer{
+                id
+                answer
+            }
+        }
+    }`
+
+export default function SubmitAnswerForm(props) {
+    const history = useHistory();
+
+    const { setVisual, questionID } = props;
+    const [answer, setAnswer] = useState("True");
+    const [articleLink, setArticleLink] = useState("");
+    const [statement, setStatement] = useState("");
+
+    const handleSubmit = () => {
+        const variables = {
+            "input": {
+                "answer": answer,
+                "text": statement,
+                "citationUrl": articleLink,
+                "citationTitle": "Hello world ctitle2",
+                "questionId": questionID
+            }
+        }
+        mutate(mutation, variables)
+            .then(() => { history.push('/login') })
+            .catch(err => console.log(err[0].message))
```

Comment:
This catch statement isn't doing much. You can add a `console.warn` in `mutate` if you'd like but I'd remove this.